x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -225,11 +241,18 @@ export class Authenticator {
       this.setupHTTPAuthenticationProvider(
         Object.freeze({
           ...providerCommonOptions,
+          name: '__http__',
```

Comment:
__note:__ have mixed feelings about this "reserved" name, but haven't managed to come with anything better yet. We can also forbid this name during config validation instead, but I'm not sure if it's worth the complexity :man_shrugging: